sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.0-beta.4 (Unreleased)
 ### Breaking Changes
+- Add `RecognizeOptions` and `RecognizeCustomFormOptions` to pass configurable options when using recognize API's on
+FormRecognizerClient.
```

Comment:
You should also add that you're switching includeTextDetails -> includeTextContent in the options class